app/src/main/res/values-sw360dp/dimens.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <dimen name="sp_12">12sp</dimen>
+    <dimen name="sp_14">14sp</dimen>
+    <dimen name="sp_16">16sp</dimen>
+    <dimen name="sp_18">18sp</dimen>
+    <dimen name="dp_0.5">0.5dp</dimen>
```

Comment:
WARNING: [UnusedResources] The resource `R.dimen.dp_0.5` appears to be unused
 Unused resources make applications larger and slow down builds.